docs/user_guide/overview.rst

```diff
@@ -26,24 +26,28 @@ Operate
 -------
 Lead scientists / administrators control the federated learning process: deploy application, check statuses, start / abort / shutdown training
 
+.. _provisioned_setup:
 
 ******************************************************************************
 Provision: Configure and generate packages for the server, clients, and admins
 ******************************************************************************
-
 One party leads the process of configuring the provisioning tool and using it to generate startup kits for each party in
 the federated learning training project:
 
 Preparation for using the provisioning tool
 ===========================================
-
 After :ref:`installation`, the provisioning tool is available via ``provision`` command.
 
 Provisioning a federated learning project
 =========================================
-The :ref:`provisioning` page has details on the contents of the
-provisioning tool. Edit the :ref:`user_guide/provisioning_tool:Project yaml file` in the directory with the provisioning tool to meet your
-project requirements, then run the startup kit with (here we assume your project.yml is in current working directory)::
+The :ref:`provisioning` page has details on the contents of the provisioning tool and the Open Provision API, which you
```

Comment:
Where do we define the "Open Provision API"?

Users might be confused when they see this.

Reply:
Added some clarification for this